mobile/v1/redshift/sql-runner/sql/tests/00-staging-reconciliation/XX-destroy/XX-destroy-staging-reconciliation.sql

```diff
@@ -0,0 +1,18 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+DROP TABLE IF EXISTS {{.scratch_schema}}.mobile_staging_reconciliation{{.entropy}};
```

Comment:
No need for both cleanup and destroy steps for this. They do the same thing, let's keep just the one.

Reply:
Fair. Removed